maestro-cli/src/main/java/maestro/cli/command/StudioCommand.kt

```diff
@@ -33,7 +33,7 @@ class StudioCommand : Callable<Int> {
             throw CliError("--platform option was deprecated. You can remove it to run your test.")
         }
 
-        MaestroSessionManager.newSession(parent?.host, parent?.port, parent?.deviceId) { session ->
+        MaestroSessionManager.newSession(parent?.host, parent?.port, parent?.deviceId, launchWebDevice = false, isStudio = true) { session ->
```

Comment:
nit: instead of `launchWebDevice` how about passing `deviceId = "chrome"` instead? This would make the API a bit more flexible